src/dispatch-payload.ts

```diff
@@ -142,3 +125,23 @@ function getFunctionCallbackID(
         ?.function_data?.function?.callback_id ?? "";
   }
 }
+
+export function extractBaseHandlerArgsFromPayload(
+  payload: InvocationPayload<ValidInvocationPayloadBody>,
+): BaseHandlerArgs {
+  const { body, context } = payload;
+  const env = context.variables || {};
+  const team_id = context.team_id || "";
+  const enterprise_id = body.enterprise_id ||
+    (body as BaseEventInvocationBody).enterprise?.id || "";
+  const token = body.event?.bot_access_token || context.bot_access_token || "";
+  const inputs = body.event?.inputs || {};
+  return {
+    body,
```

Comment:
Worth noting that prior to this PR, all events _except_ `function_executed` got a `body` argument that mapped to the payload's `body` argument. I see no reason to not simplify this and also forward the payload's `body` for the `function_executed` event, but if you disagree, speak!